admin_guide/pruning_resources.adoc

```diff
@@ -235,6 +235,16 @@ case it fails (the registry cannot be resolved or reached), an alternative
 route that works needs to be provided using this flag. The registry host name
 may be prefixed by `https://` or `http://` which will enforce particular
 connection protocol.
+
+.^|`--prune-registry`
+|In conjunction with the conditions stipulated by the other options, this option controls
+the operation only considers the {product-title} Image API Objects, or also processes
```

Comment:
`this option controls the operation only considers the `

should be "this option controls whether the"?